lib/phlex/testing/capybara.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require "capybara"
+
+module Phlex::Testing::Capybara
+	include Phlex::Testing::SGML
+
+	def self.included(mod)
+		if defined?(Minitest::Test) && Minitest::Test > mod
+			require "capybara/minitest"
+			include Capybara::Minitest::Assertions
+		end
+
+		if defined?(RSpec::Core::ExampleGroup) && RSpec::Core::ExampleGroup > mod
+			require "capybara/rspec"
+			include Capybara::RSpecMatchers
+		end
```

Comment:
Automatically detect which testing framework you’re using an include relevant matchers / assertions.